lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_japan.govspeak.erb

```diff
@@ -50,7 +50,6 @@
   <%= render partial: 'consular_fees_table.govspeak.erb',
              locals: {
                services: [
-                 :administering_oath_or_making_declaration,
```

Comment:
Should we still keep this partial here.. it appears empty.. isn't it best to leave this out.. see screenshots

## GOVUK
![screen shot 2016-03-30 at 14 16 30](https://cloud.githubusercontent.com/assets/84896/14143262/47265974-f682-11e5-8cfa-5dd485ae847c.png)


## My Localhost running this branch


![screen shot 2016-03-30 at 14 17 35](https://cloud.githubusercontent.com/assets/84896/14143258/427795f0-f682-11e5-8697-b50ab267c952.png)